tests/configfiles/DockerfileLintBearTest.py

```diff
@@ -24,6 +26,69 @@
 """
 
 
+good_file_2 = """
+FROM ubuntu
+MAINTAINER coala
+LABEL Name coala-docker
+LABEL Version 0.1
+
+RUN apt-get -y -qq update
+RUN apt-get -y -qq upgrade
+RUN curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+EXPOSE 5432
+CMD ["/usr/bin/node", "/var/www/app.js"]
+"""
+
+
+bad_file_2 = """
+FROM ubuntu
+
+apt-get -y -qq update
+apt-get -y -qq upgrade
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+"""
+
+good_file_3 = """
+FROM kali
+MAINTAINER coala
+LABEL Name coala-docker
+LABEL Version 0.1
+EXPOSE 5432
+"""
+
+
+bad_file_3 = """
```

Comment:
Optional: usually there's no underscore between a word and a number. Just `bad_file3`, `good_file2` etc is enough.